src/shadowbox/server/shared_metrics.spec.ts

```diff
@@ -20,7 +20,7 @@ import {AccessKeyConfigJson} from './server_access_key';
 
 import {ServerConfigJson} from './server_config';
 import {
-  CountryUsage,
+  LocationUsage,
```

Comment:
Why are you renaming to location if it's a country?
Should we just call it `Usage`, `ReportedUsage`, `UsageEntry`?
That may be less confusing, since we now have multiple entries for the same location.

Reply:
Technically we won't have multiple entries for the same location, since a location is a country+ASN combination, whereas we would have multiple entries per country, that's why I wanted to rename it to make that clearer.

I like your alternative suggestions though. I guess we got rid of `KeyUsage` in https://github.com/Jigsaw-Code/outline-server/pull/1529 so we really don't need to distinguish between those usages anymore. Let's go with `Usage`?